requirements.txt

```diff
@@ -1,10 +1,6 @@
 click >= 6.6
 cloudpickle >= 1.5.0
-contextvars;python_version<'3.7'
-# TODO: Update our dask version requirement to
-# the latest dask release before releasing distributed
-# xref https://github.com/dask/distributed/pull/4489
-dask @ git+https://github.com/dask/dask.git@master
+dask >= 2021.03.0
```

Comment:
Did you intend to edit this file ?  Probably should fetch latest from main

Reply:
Nope - I did this to fix local dev builds while the main/master switch was happening and committed unnecessarily - although this does seem to match the current upstream requirements?

https://github.com/dask/distributed/blob/7c6709fd8d2ee6356ba43ceff64cd04e9803b2a1/requirements.txt#L1-L4

Should I change this to pull the latest from the main branch?

Reply:
Yeah, I would pull from latest in main.